Signal/src/call/PeerConnectionClient.swift

```diff
@@ -393,30 +400,42 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
                 Logger.debug("\(strongSelf.logTag) \(#function) Ignoring obsolete event in terminated client")
                 return
             }
-            guard let localVideoTrack = strongSelf.localVideoTrack else {
+
+            guard let videoCaptureController = strongSelf.videoCaptureController else {
                 Logger.debug("\(strongSelf.logTag) \(#function) Ignoring obsolete event in terminated client")
                 return
             }
-            guard let videoCaptureSession = strongSelf.videoCaptureSession else {
+
+            guard let localVideoTrack = strongSelf.localVideoTrack else {
                 Logger.debug("\(strongSelf.logTag) \(#function) Ignoring obsolete event in terminated client")
                 return
             }
-
             localVideoTrack.isEnabled = enabled
 
             if enabled {
-                Logger.debug("\(strongSelf.logTag) in \(#function) starting videoCaptureSession")
-                videoCaptureSession.startRunning()
+                Logger.debug("\(strongSelf.logTag) in \(#function) starting video capture")
+                videoCaptureController.startCapture()
             } else {
-                Logger.debug("\(strongSelf.logTag) in \(#function) stopping videoCaptureSession")
-                videoCaptureSession.stopRunning()
+                Logger.debug("\(strongSelf.logTag) in \(#function) stopping video capture")
+                videoCaptureController.stopCapture()
             }
 
             DispatchQueue.main.async(execute: completion)
         }
     }
 
-    // MARK: Audio
+    // MARK: VideoCaptureSettingsDelegate
+
+    // MJK: fixme
+    var videoWidth: Int32 {
```

Comment:
Default width/height is TBD, but if you'd like to review the remaining changes, I could let you know which values I settle on.

Reply:
After some testing, this actually seems like a reasonable default. I'm checking with Android to make sure we're consistent.